lib/content-services/src/lib/search/components/search-radio/search-radio.component.html

```diff
@@ -6,10 +6,7 @@
         *ngFor="let option of options"
         [value]="option.value"
         class="adf-facet-filter">
-        <div
-            matTooltip="{{ option.name | translate }}"
-            matTooltipPosition="right"
```

Comment:
Same here

Reply:
it's the same behavior as before but without extra lib